drivers/sensor/lsm6dsl/lsm6dsl.h

```diff
@@ -667,9 +667,6 @@ struct lsm6dsl_data {
 #endif
 	const struct lsm6dsl_transfer_function *hw_tf;
 	uint16_t accel_freq;
-	uint8_t accel_fs;
-	uint16_t gyro_freq;
-	uint8_t gyro_fs;
```

Comment:
squash this commit with the one with accel_freq

Reply:
Well, those are two separate logical changes:
- remove unused `accel_fs` and `gyro_{fs,freq}`
- drop double assign of `accel_freq`

and there are reasons provided for each commit. This also makes it much easier to fit into commit title character limits.

Reply:
ok, agreed